ksqldb-engine/src/main/java/io/confluent/ksql/planner/LogicalPlanner.java

```diff
@@ -631,11 +633,9 @@ private JoinNode buildJoin(final Join root, final String prefix, final boolean i
    * @return whether this is a foreign key join or not
    */
   // CHECKSTYLE_RULES.OFF: CyclomaticComplexity
-  private boolean verifyJoin(
-      final JoinInfo joinInfo,
-      final PlanNode leftNode,
-      final PlanNode rightNode
-  ) {
+  private boolean verifyJoin(final JoinInfo joinInfo,
```

Comment:
nit: the existing spacing pattern is consistent with what's used throughout most of the ksqlDB code base. This change is moving away from the norm.